test/system/070-build.bats

```diff
@@ -1097,6 +1097,18 @@ EOF
     run_podman rmi -f build_test
 }
 
+@test "podman build empty context dir" {
+    tmpdir=$PODMAN_TMPDIR/emptydir
```

Comment:
`tmpdir` is such a dangerous variable name; something like `emptydir` or `contextdir` would be more intuitive to a maintainer.

Reply:
I agree. However `tmpdir` is used everywhere in this file. Do you want me to change only this newly added `tmpdir` or all of them in this file ? I am asking because different maintainers have different rules, for example in my projects usually I require these changes of names to be done separately, while others are ok to have it done in one commit.

Reply:
Ugh, you're right, and several of those are mine. So, in the interests of consistency, you're welcome to leave it as `tmpdir` and I'll do a cleanup PR later. Or you're welcome to improve the code base by setting a better example in your added test (but no, please don't clean up the others, that is far beyond the scope of this PR). Your choice, and either is fine.

Reply:
Ok, I have chosen the name `contextdir`, hope this is ok.

Reply:
Ah I jsut saw there is already `PODMAN_contextdir` (which is the parent folder of `contextdir`). So this name might be confusing too. Maybe I will change to `buildcontextdir` instead.